docs/ui/components/Markdown/index.tsx

```diff
@@ -6,7 +6,9 @@ import { Blockquote } from './Blockquote';
 
 import { DETAILS, SUMMARY } from '~/ui/components/Collapsible';
 import { Cell, HeaderCell, Row, Table, TableHead } from '~/ui/components/Table';
-import { A, H1, H2, H4, H5, CODE, P, BOLD, UL, OL, LI } from '~/ui/components/Text';
+import { A, CODE, P, BOLD, UL, OL, LI, createTextComponent } from '~/ui/components/Text';
+import { TextElement } from '~/ui/components/Text/types';
+import { withAnchor } from '~/ui/components/Text/withAnchor';
```

Comment:
I would move this over from `~/ui/components/Text` to `~/ui/components/Markdown` (or even a new `components/Anchor` folder 😄

Reply:
That's good idea, I will move the Anchor in the future PR. 🙂